ci/sync_from_upstream_tf.sh

```diff
@@ -27,28 +27,21 @@ SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 ROOT_DIR=${SCRIPT_DIR}/..
 cd "${ROOT_DIR}"
 
-rm -rf /tmp/tensorflow
-
-git clone https://github.com/tensorflow/tensorflow.git --depth=1 /tmp/tensorflow
-
 # As part of the import from upstream TF, we generate the Python bindings for
 # the TfLite flatbuffer schema.
 
-# TfLite is using flatbuffers2.0 but TFLM is currently staying with 1.12.0
-# See http://b/235888271 for additional context as well as some of the workflows
-# that we will need to fix before upgrading to a newer flatbuffer version.
-rm -rf /tmp/tensorflow/third_party/flatbuffers
-cp -r ci/flatbuffers_for_tf_sync/ /tmp/tensorflow/third_party/flatbuffers
-
-cd /tmp/tensorflow
-bazel build tensorflow/lite/python:schema_py
-/bin/cp bazel-bin/tensorflow/lite/python/schema_py_generated.py tensorflow/lite/python
+bazel build tensorflow/lite/python:schema_py_generate
+rm -rf tensorflow/lite/python/schema_py_generated.py
```

Comment:
let's make a bug and put in a TODO(b/) to address whether the rm is really needed or not.